src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -57,22 +59,45 @@ def index(self):
         # TODO: remove this method
         return self._migration_status_refresher.index()
 
-    def migrate_tables(self, what: What, acl_strategy: list[AclMigrationWhat] | None = None):
+    def migrate_tables(
+        self,
+        what: What,
+        acl_strategy: list[AclMigrationWhat] | None = None,
+        hiveserde_in_place_migrate: HiveSerdeType | None = None,
+        mounts_crawler: Mounts | None = None,
```

Comment:
why do we have to pass it all the way down across 3 methods, when we should inject it via constructor?

Reply:
yes, it will be updated in coming commits